src/xv.c

```diff
@@ -214,7 +214,6 @@ int main(int argc, char **argv)
 
   picComments = (char *) NULL;
 
-  if (picExifInfo) free(picExifInfo);
   picExifInfo = (byte *) NULL;
```

Comment:
This section is the initial part if main(), which is why the other variables are set to NULL without checking to free first.
Somehow this gets lucky that picExifInfo global variable is initialized to 0. I think that C does not require that.